### PR TITLE
Add test for schur(Tridiagonal)

### PR DESCRIPTION
**Pull request by kshyatt**

### CODE REVIEW

File: test/linalg/schur.jl [hunk 1]
```diff
@@ -42,6 +42,8 @@ aimg  = randn(n,n)/2
         @test vecs*sch*vecs' ≈ asym
         sch, vecs, vals = schur(Symmetric(a+a.'))
         @test vecs*sch*vecs' ≈ a + a.'
+        sch, vecs, vals = schur(Tridiagonal(a+a.'))
+        @test vecs*sch*vecs' ≈ AbstractArray(Tridiagonal(a + a.'))
```
Comment (Sacha0) [inline test/linalg/schur.jl]: The `AbstractArray` seems unnecessary? :)

Comment (kshyatt) [reply test/linalg/schur.jl]: Is it? I think the types won't match? Or are you saying isapprox will do the convert for us?

Comment (Sacha0) [reply test/linalg/schur.jl]: The latter :).

Comment (kshyatt) [reply test/linalg/schur.jl]: Nice, I'll edit it then. Thanks for catching that!

**Comment by kshyatt** [comment]
@Sacha0 good to go if CI passes?

**Comment by Sacha0** [comment]
Thanks @kshyatt! :)

**Pull request merged**